src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -485,6 +494,88 @@ public void sendEmailWhenOrderIsServed(String authorisation,
 
     }
 
+    private void sendEmailToOtherOrganisation(CaseData caseData, List<Element<EmailInformation>> emailInformationCA,
+                                              String authorisation, List<Document> orderDocuments) {
+
+        Map<String, Object> dynamicData = EmailUtils.getCommonSendgridDynamicTemplateData(caseData);
+
+        if (null != caseData.getManageOrders() && null != caseData.getManageOrders().getServeOrderDynamicList()) {
+            List<String> selectedOrderIds = caseData.getManageOrders().getServeOrderDynamicList().getValue()
+                .stream().map(DynamicMultiselectListElement::getCode).toList();
+            AtomicBoolean newOrdersExists = new AtomicBoolean(false);
```

Comment:
Since it is local variable and each thread will have its own copy . hence Atomic should not be used here.